packages/plugin-react/src/index.ts

```diff
@@ -23,6 +23,14 @@ export interface Options {
    * @default "classic"
    */
   jsxRuntime?: 'classic' | 'automatic'
+  /**
+   * Replaces the import source when importing functions when used with automatic jsx runtime.
+   *
+   * See https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#importsource
+   * @default "react"
+   */
+  jsxImportSource?: string
```

Comment:
Is there actually a use case for exposing this option?

Reply:
Yes. e.g., if I want to use `jsx` from `'@emotion/react'` instead from `'react'`, I will use
```js
    reactPlugin({
      jsxRuntime: 'automatic',
      jsxImportSource: '@emotion/react'
    })
```
see https://emotion.sh/docs/css-prop#:~:text=configuration-,babelrc,-presets